components/sql/src/test/scala/pl/touk/nussknacker/sql/service/DatabaseLookupEnricherTest.scala

```diff
@@ -76,7 +76,7 @@ class DatabaseLookupEnricherTest extends BaseHsqlQueryEnricherTest {
     val result: service.TransformationStepResult = definition(service.TransformationStep(List(), None))
     result match {
       case service.NextParameters(parameters, _, _) =>
-        parameters.head.editor.get shouldBe FixedValuesParameterEditor(
+        parameters.head.editors.head.mainEditor shouldBe FixedValuesParameterEditor(
```

Comment:
Please use `.get` instead of `.head`. With .head it looks like you'd have list of lists

Reply:
Done